### PR TITLE
inline certain font assets

### PR DESCRIPTION
**Pull request by josh-m-sharpe**
The goal of this PR is to inline certain font assets which eliminates an additional HTTP request.  The gem depends on sass (by way of sass-rails) so, this should be compatible.

I've only inlined the eot, woff and woff2 files as they are the most likely to be used and also the smallest.  I didn't think it made sense to inline the svg/ttf files since (and I'm just guessing) they're the least likely to be used.  They're also fairly large and would incur another half a meg of unzipped content added to the asset package

FWIW, this can all be accomplished by overriding @font-face in an application-level css file -- however, I thought this might be a useful improvement to the gem.  Open to discussing.  Thanks!

### CODE REVIEW

**Comment by rmm5t** [comment]
>  The gem depends on sass (by way of sass-rails) so, this should be compatible.

No, sorry, **this gem does not depend on sass-rails**. Solely depending on only `railties` actually one of it's underlying design goals. Yes, it's compatible with sass-rails, but it does not rely on it in any way.

> I've only inlined the eot, woff and woff2 files as they are the most likely to be used

While true for modern browsers, TTF and SVG are never downloaded if the browser doesn't prefer that format.  Only one font format is ever downloaded per browser, and the browser can cache this font after the first download. **So, with the changes in this proposed pull-request we've now increased the payload by 4 times** (3 fonts downloaded instead of just 1, and 4/3 the original size for each (because the fonts need to be converted to base64 first...not considering HTTP compression schemes here).

In summary, this proposed change forces the browser to download more data than necessary. I would highly doubt that eliminating that extra HTTP request yields any appreciable improvement. In fact, it likely hurts things on the first request -- especially if the download rate is relatively slow (e.g. on mobile).
